test/benchmark/bench_linux_and_aster.sh

```diff
@@ -49,15 +67,26 @@ run_benchmark() {
     local linux_output="${BENCHMARK_DIR}/linux_output.txt"
     local aster_output="${BENCHMARK_DIR}/aster_output.txt"
     local result_template="${BENCHMARK_DIR}/${benchmark}/result_template.json"
-    local result_file="result_${benchmark}.json"
-
-    # Entrypoint script for initramfs
-    local initramfs_entrypoint_script="${BENCHMARK_DIR}/benchmark_entrypoint.sh"
-    generate_entrypoint_script "${benchmark}" > "${initramfs_entrypoint_script}"
-    chmod +x "${initramfs_entrypoint_script}"
-        
-    # TODO: enable nopti for Linux to make the comparison more fair
-    local qemu_cmd="/usr/local/qemu/bin/qemu-system-x86_64 \
+    local benchmark_name=$(basename "${benchmark}")
+    local result_file="result_${benchmark_name}.json"
+    
+    echo "Preparing libraries..."
+    prepare_libs
+
+    # OSTD_TASK_STACK_SIZE_IN_PAGES=7 This is for better fork.
+    # # TODO: remove this workaround after enabling kernel virtual area.
+    local asterinas_cmd="export OSTD_TASK_STACK_SIZE_IN_PAGES=7 && \
+        make build && \
+        cargo osdk run \
```

Comment:
Giving the demand that developers want to run a benchmark on Asterinas alone, we should keep the benchmark arguments in the Makefile.

I want to make a more general startup process for both Asterinas and Linux. Using `cargo osdk run` shows the `init-args` and informs developers that Asterinas and Linux share the same entrypoint script and they can add prerequisites there. 

Ideally, we should boot Asterinas and Linux with the same qemu commands, except the `-kernel`. So we can set up the arguments like port forwarding rules easier. Unfortunately, it seems that we cannot do it now. Asterinas is booted from `-drive file=/root/asterinas/target/osdk/aster-nix/aster-nix-osdk-bin.iso`. This can be done in later PR.